test/system/700-play.bats

```diff
@@ -651,3 +651,62 @@ spec:
     run_podman pod ps
     run_podman rmi $(pause_image)
 }
+
+@test "podman kube play with configmaps" {
+    skip_if_remote "the configmap argument is supported only locally"
+
+    configmap_file=${PODMAN_TMPDIR}/play_kube_configmap_configmaps$(random_string 6).yaml
+    echo "
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: foo
+data:
+  value: foo
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: bar
+data:
+  value: bar
+" > $configmap_file
+
+    pod_file=${PODMAN_TMPDIR}/play_kube_configmap_pod$(random_string 6).yaml
+    echo "
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  restartPolicy: Never
+  containers:
+  - name: server
+    image: $IMAGE
+    env:
+    - name: FOO
+      valueFrom:
+        configMapKeyRef:
+          name: foo
+          key: value
+    - name: BAR
+      valueFrom:
+        configMapKeyRef:
+          name: bar
+          key: value
+    command:
+    - /bin/sh
+    args:
+    - -c
+    - "echo \$FOO:\$BAR"
+" > $pod_file
+
+    run_podman kube play --configmap=$configmap_file $pod_file
+    run_podman logs test_pod-server
```

Comment:
You need to do a `podman wait` in between to prevent flakes, play just start we do not know if the echo command was executed by the time we run logs. We actually have to make sure the container exited before running logs.
cc @edsantiago

Reply:
Done